### DISCUSSION TITLE
[Feature] : Adding contributors section to the README.md file.

### DISCUSSION THREAD

**Discussion by Kalyanimhala** [immich-app/immich#4681] [Feature Request]
### The feature

There is no Contributors section in readme file .
As we know Contributions are what make the open-source community such an amazing place to learn, inspire, and create.
The Contributors section in a README.md file is important as it acknowledges and gives credit to those who have contributed to a project, fosters community and collaboration, adds transparency and accountability, and helps document the project's history for current and future maintainers. It also serves as a form of recognition, motivating contributors to continue their efforts.
![contributors](https://github.com/immich-app/immich/assets/147402107/e623a2e6-a13e-44e6-a423-1cbe35f80661)

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by Kalyanimhala** [comment]
@alextran1502  Kindly assign me this issue , I want to work on it, Thank You!!

**Reply by alextran1502** [reply]
Please go ahead

**Comment by jrasm91** [comment]
Seems a little bit less important since this functionality is built into GitHub automatically and doesn't need to be synced and updated in a separate file.
